deps/llvm.mk

```diff
@@ -336,6 +336,7 @@ endif
 ifneq ($(LLVM_LLDB_TAR),)
 	$(JLCHECKSUM) $(LLVM_LLDB_TAR)
 endif
+	-rm -rf $(LLVM_SRC_DIR)
```

Comment:
including on travis where merging this would delete the cached llvm and occupy travis for several hours